include/inviwo/core/datastructures/datamapper.h

```diff
@@ -51,15 +51,37 @@ class DataFormatBase;
  */
 class IVW_CORE_API DataMapper {
 public:
+    /**
+     * Signed fixed-point integers are normalized using either an asymmetric range
+     * [-2^(b-1), 2^(b-1) - 1] or a symmetric range [-2^(b-1) + 1, 2^(b-1) - 1], where b
+     * corresponds to the bit depth of the fixed-point integer.
+     *
+     * From version 4.2 and above, OpenGL uses a symmetric range to convert between normalized
```

Comment:
Should/can the setting be retrieved from the application when one knows the OpenGL version?

Reply:
See addition in OpenGLCapabilities.

https://github.com/inviwo/inviwo/blob/1315b02b2ed92bc21bf48310e6609f748e2272eb/modules/opengl/include/modules/opengl/openglcapabilities.h#L96-L104

Reply:
The shaderutils responsible for data range scaling and offset use this setting to use the correct version in connection with the sign_normalized GL formats.